src/components/reservation/ReservationFormListItem.tsx

```diff
@@ -8,6 +8,7 @@ interface ReservationFormListItemProps {
   subText?: string;
   changeHandler?: () => void;
   buttonAriaLabel?: string;
+  isPossibleToChangeReservationDetails?: boolean | null;
```

Comment:
Consider if it is in fact necessary to both allow the prop to be undefined *and* passing `null`.